server.js

```diff
@@ -36,6 +35,27 @@ function connectToDb() {
    });
 }
 
+function findUser(db, nickname, tag) {
+   return new Promise(resolve => {
+      db.collection('accounts').findOne({ nickname: `${nickname}`, tag: tag }, (err, result) => {
+         if (err || !result) {
+            console.log("User not found");
+            return resolve(null);
+         }
+         return resolve(result._id);
```

Comment:
Instead of returning `result._id` I would return `result.uniqid`. `_id` is mongodb's internal primary key and it will change if we ever migrate databases or something, `uniqid` is our own unique user id that should never change so it's a little safer

You'd have to change adding user to friend requests too
```js
db.collection('accounts').updateOne({ "uniqid": userGettingRequest }, { ...
```